### ISSUE TITLE
can you add presets?

### ISSUE THREAD

**Issue by skypce** [pmarks-net/chromadoze#14]
Hi it it is very good, but will be great if you add preset for basic usage.

**Comment by callis** [comment]
I would love to see the feature too. E.g. white/pink/red/grey noise

**Comment by Rbagman** [comment]
Brown too

**Comment by cosmopol** [comment]
I totally agree. I wonder if there is a quick way of extracting the tonal features of a given set of naturally occurring soundscapes? Naming, description and maybe a sound grid resolution increase needed to be implemented. I could assist, if you want me to.

**Comment by cosmopol** [comment]
BTW: I love this app :-)

**Comment by pataquets** [comment]
Yes please. White/brown/pink noise presets are absolutely needed.
Other than that, awesome app! Thanks for sharing.

**Comment by delight-f** [comment]
Please consider adding some basic presets!
